src/mappings/mappingHandlers.ts

```diff
@@ -40,6 +42,22 @@ BigInt.prototype.toJSON = function () {
   return this.toString();
 };
 
+const API_ENDPOINT = 'https://main-a.api.agoric.net:443'
```

Comment:
There is a parameter in project.js called network which has an endpoint URL list. Project.js is complied into project.yaml. Is there a was way to read network.endpoint list instead of creating another variable here?